src/components/JobLogsContainer/JobLogsContainer.js

```diff
@@ -86,8 +86,9 @@ const JobLogsContainer = props => {
   const getFileNameField = record => {
     const fileName = get(record.exportedFiles, '0.fileName');
     const isSuccessfulRecord = record.status === JOB_EXECUTION_STATUSES.SUCCESS;
+    const areAllJobsFailed = !record.progress.exported;
 
-    if (!isSuccessfulRecord) return fileName;
+    if (!isSuccessfulRecord || areAllJobsFailed) return fileName;
```

Comment:
We need remove the `!isSuccessfulRecord` check as in this case the Scenario 1 won't
```suggestion
    if (areAllJobsFailed) return fileName;
```

Reply:
It looks logical, thanks!